common/app/views/fragments/footer.scala.html

```diff
@@ -76,211 +76,298 @@
                     </div>
                 }
 
-            <ul class="colophon__list">
+            <div class="colophon__lists-container">
```

Comment:
None of the links changed, but I moved the markup around quite a lot. Sorry for confusing diffs 😞